src/blockdev.rs

```diff
@@ -212,7 +212,7 @@ impl PartTableKernel {
 
 impl PartTable for PartTableKernel {
     fn reread(&mut self) -> Result<()> {
-        reread_partition_table(&mut self.file)?;
+        reread_partition_table(&mut self.file, NonZeroU32::new(20).unwrap())?;
```

Comment:
Since this is the usual case, maybe we can define a `DEFAULT_PARTITION_TABLE_RETRIES`?  Or, to simplify, it might make sense to just have the function take a `retry` boolean.